concepts/frontend.md

```diff
@@ -1,3 +1,25 @@
 # The Xing Frontend
 
-Xing is built on AngularJS, which is a nifty, well supported SPA framework in its own right. We incorporated a ES6 build process, as well as ES-future decorators to speed the development process and improve code structure.
+Xing is built on AngularJS, the well-supported SPA framework by Google.  We incorporated a ES6 build process, as well as ES-future decorators to speed the development process and improve code structure.
+
+## ES6 / ESNext
+
+Xing uses current generation JavaScript, and includes an integrated build environment with [Traceur](https://github.com/google/traceur-compiler) to provide transpilation.
```

Comment:
I would say "next generation Javascript" -- browsers do not support the type of code we write (without transpilation) so I would say ES5 is still effectively the "current" generation of Javascript"